packages/voting/graphql/resolvers/Candidacy.js

```diff
@@ -18,12 +18,53 @@ module.exports = {
       return
     }
 
-    if (!candidacy.user.address.city) {
+    if (!candidacy.user.address?.city) {
       return
     }
 
     return candidacy.user.address.city
   },
+  postalCodeGeo: async (candidacy, args, { user: me, pgdb }) => {
+    if (!Roles.userIsInRoles(me, ['admin', 'supporter', 'associate'])) {
+      return
+    }
+
+    if (
+      !candidacy.user.address?.postalCode &&
+      !candidacy.user.address?.country
+    ) {
+      return
+    }
+
+    const { postalCode, country } = candidacy.user.address
+
+    const geoResult = await pgdb.queryOne(
+      `
+      SELECT 
+        c.code, 
+        pc.lat, 
+        pc.lon
+      FROM "statisticsGeoCountry" c
+      JOIN "statisticsGeoPostalCode" pc
+      ON c."code" = pc."countryCode"
+      WHERE 
+        c."name" = :country AND 
+        pc."postalCode" = :postalCode
+    `,
+      {
+        country,
+        postalCode,
+      },
+    )
```

Comment:
Due to amount of queries at once, this is likely an ideal candidate for a data loader.